[2022-12-24 22:29:43,386] {processor.py:153} INFO - Started process (PID=1377) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:29:43,387] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:29:43,388] {logging_mixin.py:115} INFO - [2022-12-24 22:29:43,387] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:29:43,421] {logging_mixin.py:115} INFO - [2022-12-24 22:29:43,406] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 41
    retries = 3,
          ^
SyntaxError: invalid syntax
[2022-12-24 22:29:43,422] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:29:43,442] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.058 seconds
[2022-12-24 22:30:13,612] {processor.py:153} INFO - Started process (PID=1387) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:30:13,613] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:30:13,613] {logging_mixin.py:115} INFO - [2022-12-24 22:30:13,613] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:30:13,633] {logging_mixin.py:115} INFO - [2022-12-24 22:30:13,632] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 41
    retries = 3,
          ^
SyntaxError: invalid syntax
[2022-12-24 22:30:13,633] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:30:13,661] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.052 seconds
[2022-12-24 22:30:43,861] {processor.py:153} INFO - Started process (PID=1397) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:30:43,862] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:30:43,863] {logging_mixin.py:115} INFO - [2022-12-24 22:30:43,863] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:30:43,889] {logging_mixin.py:115} INFO - [2022-12-24 22:30:43,888] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 41
    retries = 3,
          ^
SyntaxError: invalid syntax
[2022-12-24 22:30:43,890] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:30:43,969] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.118 seconds
[2022-12-24 22:30:47,907] {processor.py:153} INFO - Started process (PID=1402) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:30:47,908] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:30:47,908] {logging_mixin.py:115} INFO - [2022-12-24 22:30:47,908] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:30:47,960] {logging_mixin.py:115} INFO - [2022-12-24 22:30:47,955] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:30:47,963] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:30:47,985] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.082 seconds
[2022-12-24 22:31:18,173] {processor.py:153} INFO - Started process (PID=1412) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:31:18,174] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:31:18,174] {logging_mixin.py:115} INFO - [2022-12-24 22:31:18,174] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:31:18,200] {logging_mixin.py:115} INFO - [2022-12-24 22:31:18,196] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:31:18,203] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:31:18,219] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.049 seconds
[2022-12-24 22:31:48,389] {processor.py:153} INFO - Started process (PID=1422) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:31:48,390] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:31:48,390] {logging_mixin.py:115} INFO - [2022-12-24 22:31:48,390] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:31:48,412] {logging_mixin.py:115} INFO - [2022-12-24 22:31:48,408] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:31:48,414] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:31:48,440] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.053 seconds
[2022-12-24 22:32:18,608] {processor.py:153} INFO - Started process (PID=1432) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:32:18,609] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:32:18,610] {logging_mixin.py:115} INFO - [2022-12-24 22:32:18,610] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:32:18,637] {logging_mixin.py:115} INFO - [2022-12-24 22:32:18,631] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:32:18,639] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:32:18,655] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.050 seconds
[2022-12-24 22:32:48,823] {processor.py:153} INFO - Started process (PID=1442) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:32:48,824] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:32:48,824] {logging_mixin.py:115} INFO - [2022-12-24 22:32:48,824] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:32:48,847] {logging_mixin.py:115} INFO - [2022-12-24 22:32:48,843] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:32:48,849] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:32:48,876] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.055 seconds
[2022-12-24 22:33:19,060] {processor.py:153} INFO - Started process (PID=1452) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:33:19,061] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:33:19,062] {logging_mixin.py:115} INFO - [2022-12-24 22:33:19,061] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:33:19,085] {logging_mixin.py:115} INFO - [2022-12-24 22:33:19,080] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:33:19,087] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:33:19,115] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.057 seconds
[2022-12-24 22:33:49,300] {processor.py:153} INFO - Started process (PID=1462) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:33:49,301] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:33:49,302] {logging_mixin.py:115} INFO - [2022-12-24 22:33:49,301] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:33:49,325] {logging_mixin.py:115} INFO - [2022-12-24 22:33:49,321] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:33:49,327] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:33:49,353] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.056 seconds
[2022-12-24 22:34:19,523] {processor.py:153} INFO - Started process (PID=1472) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:34:19,524] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:34:19,525] {logging_mixin.py:115} INFO - [2022-12-24 22:34:19,525] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:34:19,548] {logging_mixin.py:115} INFO - [2022-12-24 22:34:19,544] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:34:19,550] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:34:19,564] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.044 seconds
[2022-12-24 22:34:49,779] {processor.py:153} INFO - Started process (PID=1482) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:34:49,779] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:34:49,780] {logging_mixin.py:115} INFO - [2022-12-24 22:34:49,780] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:34:49,802] {logging_mixin.py:115} INFO - [2022-12-24 22:34:49,798] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:34:49,804] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:34:49,820] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.044 seconds
[2022-12-24 22:35:19,991] {processor.py:153} INFO - Started process (PID=1492) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:35:19,992] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:35:19,992] {logging_mixin.py:115} INFO - [2022-12-24 22:35:19,992] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:35:20,017] {logging_mixin.py:115} INFO - [2022-12-24 22:35:20,013] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:35:20,019] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:35:20,037] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.048 seconds
[2022-12-24 22:35:50,204] {processor.py:153} INFO - Started process (PID=1502) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:35:50,205] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:35:50,205] {logging_mixin.py:115} INFO - [2022-12-24 22:35:50,205] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:35:50,228] {logging_mixin.py:115} INFO - [2022-12-24 22:35:50,223] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:35:50,230] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:35:50,246] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.045 seconds
[2022-12-24 22:36:20,416] {processor.py:153} INFO - Started process (PID=1512) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:36:20,416] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:36:20,417] {logging_mixin.py:115} INFO - [2022-12-24 22:36:20,417] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:36:20,441] {logging_mixin.py:115} INFO - [2022-12-24 22:36:20,436] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:36:20,442] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:36:20,469] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.056 seconds
[2022-12-24 22:36:50,643] {processor.py:153} INFO - Started process (PID=1522) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:36:50,644] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:36:50,645] {logging_mixin.py:115} INFO - [2022-12-24 22:36:50,645] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:36:50,671] {logging_mixin.py:115} INFO - [2022-12-24 22:36:50,666] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:36:50,673] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:36:50,701] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.060 seconds
[2022-12-24 22:37:20,878] {processor.py:153} INFO - Started process (PID=1532) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:37:20,878] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:37:20,879] {logging_mixin.py:115} INFO - [2022-12-24 22:37:20,879] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:37:20,907] {logging_mixin.py:115} INFO - [2022-12-24 22:37:20,902] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"]
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:37:20,908] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:37:20,937] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.062 seconds
[2022-12-24 22:37:48,105] {processor.py:153} INFO - Started process (PID=1542) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:37:48,105] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:37:48,106] {logging_mixin.py:115} INFO - [2022-12-24 22:37:48,106] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:37:48,157] {logging_mixin.py:115} INFO - [2022-12-24 22:37:48,151] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:37:48,159] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:37:48,190] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.088 seconds
[2022-12-24 22:38:18,377] {processor.py:153} INFO - Started process (PID=1552) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:38:18,378] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:38:18,379] {logging_mixin.py:115} INFO - [2022-12-24 22:38:18,378] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:38:18,402] {logging_mixin.py:115} INFO - [2022-12-24 22:38:18,397] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:38:18,403] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:38:18,431] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.056 seconds
[2022-12-24 22:38:48,595] {processor.py:153} INFO - Started process (PID=1562) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:38:48,595] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:38:48,596] {logging_mixin.py:115} INFO - [2022-12-24 22:38:48,596] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:38:48,621] {logging_mixin.py:115} INFO - [2022-12-24 22:38:48,617] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:38:48,623] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:38:48,640] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.048 seconds
[2022-12-24 22:39:18,817] {processor.py:153} INFO - Started process (PID=1572) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:39:18,817] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:39:18,818] {logging_mixin.py:115} INFO - [2022-12-24 22:39:18,818] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:39:18,844] {logging_mixin.py:115} INFO - [2022-12-24 22:39:18,839] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:39:18,846] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:39:18,875] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.061 seconds
[2022-12-24 22:39:34,974] {processor.py:153} INFO - Started process (PID=1577) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:39:34,975] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:39:34,976] {logging_mixin.py:115} INFO - [2022-12-24 22:39:34,976] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:39:35,027] {logging_mixin.py:115} INFO - [2022-12-24 22:39:35,021] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 29, in <module>
    catchup=False,
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:39:35,030] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:39:35,050] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.079 seconds
[2022-12-24 22:40:05,295] {processor.py:153} INFO - Started process (PID=1587) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:40:05,295] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:40:05,296] {logging_mixin.py:115} INFO - [2022-12-24 22:40:05,296] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:40:05,324] {logging_mixin.py:115} INFO - [2022-12-24 22:40:05,319] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 29, in <module>
    catchup=False,
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:40:05,326] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:40:05,355] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.062 seconds
[2022-12-24 22:40:35,544] {processor.py:153} INFO - Started process (PID=1597) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:40:35,545] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:40:35,545] {logging_mixin.py:115} INFO - [2022-12-24 22:40:35,545] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:40:35,574] {logging_mixin.py:115} INFO - [2022-12-24 22:40:35,569] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 29, in <module>
    catchup=False,
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:40:35,577] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:40:35,604] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.063 seconds
[2022-12-24 22:40:56,742] {processor.py:153} INFO - Started process (PID=1607) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:40:56,743] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:40:56,744] {logging_mixin.py:115} INFO - [2022-12-24 22:40:56,744] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:40:56,786] {logging_mixin.py:115} INFO - [2022-12-24 22:40:56,781] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:40:56,788] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:40:56,806] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.065 seconds
[2022-12-24 22:41:26,990] {processor.py:153} INFO - Started process (PID=1617) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:41:26,990] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:41:26,991] {logging_mixin.py:115} INFO - [2022-12-24 22:41:26,991] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:41:27,019] {logging_mixin.py:115} INFO - [2022-12-24 22:41:27,014] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:41:27,021] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:41:27,040] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.052 seconds
[2022-12-24 22:41:50,180] {processor.py:153} INFO - Started process (PID=1627) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:41:50,181] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:41:50,181] {logging_mixin.py:115} INFO - [2022-12-24 22:41:50,181] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:41:50,219] {logging_mixin.py:115} INFO - [2022-12-24 22:41:50,215] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:41:50,221] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:41:50,247] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.070 seconds
[2022-12-24 22:42:20,418] {processor.py:153} INFO - Started process (PID=1637) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:42:20,419] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:42:20,420] {logging_mixin.py:115} INFO - [2022-12-24 22:42:20,420] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:42:20,443] {logging_mixin.py:115} INFO - [2022-12-24 22:42:20,439] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:42:20,445] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:42:20,461] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.045 seconds
[2022-12-24 22:42:31,475] {processor.py:153} INFO - Started process (PID=1642) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:42:31,475] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:42:31,476] {logging_mixin.py:115} INFO - [2022-12-24 22:42:31,476] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:42:31,516] {logging_mixin.py:115} INFO - [2022-12-24 22:42:31,511] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:42:31,518] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:42:31,547] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.075 seconds
[2022-12-24 22:43:01,728] {processor.py:153} INFO - Started process (PID=1652) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:43:01,729] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:43:01,729] {logging_mixin.py:115} INFO - [2022-12-24 22:43:01,729] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:43:01,756] {logging_mixin.py:115} INFO - [2022-12-24 22:43:01,751] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:43:01,758] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:43:01,789] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.064 seconds
[2022-12-24 22:43:31,987] {processor.py:153} INFO - Started process (PID=1662) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:43:31,988] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:43:31,989] {logging_mixin.py:115} INFO - [2022-12-24 22:43:31,989] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:43:32,026] {logging_mixin.py:115} INFO - [2022-12-24 22:43:32,019] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:43:32,029] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:43:32,053] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.070 seconds
[2022-12-24 22:44:02,263] {processor.py:153} INFO - Started process (PID=1672) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:44:02,265] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:44:02,268] {logging_mixin.py:115} INFO - [2022-12-24 22:44:02,268] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:44:02,305] {logging_mixin.py:115} INFO - [2022-12-24 22:44:02,300] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:44:02,307] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:44:02,336] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.088 seconds
[2022-12-24 22:44:32,536] {processor.py:153} INFO - Started process (PID=1682) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:44:32,537] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:44:32,537] {logging_mixin.py:115} INFO - [2022-12-24 22:44:32,537] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:44:32,566] {logging_mixin.py:115} INFO - [2022-12-24 22:44:32,561] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/tutorial.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial.py", line 30, in <module>
    tags=["example"],
TypeError: __init__() got an unexpected keyword argument 'schedule'
[2022-12-24 22:44:32,569] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:44:32,594] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.061 seconds
[2022-12-24 22:44:42,679] {processor.py:153} INFO - Started process (PID=1692) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:44:42,680] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:44:42,681] {logging_mixin.py:115} INFO - [2022-12-24 22:44:42,681] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:44:42,722] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:44:42,843] {logging_mixin.py:115} INFO - [2022-12-24 22:44:42,843] {manager.py:508} INFO - Created Permission View: can read on DAG:tutorial
[2022-12-24 22:44:42,854] {logging_mixin.py:115} INFO - [2022-12-24 22:44:42,854] {manager.py:508} INFO - Created Permission View: can delete on DAG:tutorial
[2022-12-24 22:44:42,861] {logging_mixin.py:115} INFO - [2022-12-24 22:44:42,861] {manager.py:508} INFO - Created Permission View: can edit on DAG:tutorial
[2022-12-24 22:44:42,862] {logging_mixin.py:115} INFO - [2022-12-24 22:44:42,862] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:44:42,870] {logging_mixin.py:115} INFO - [2022-12-24 22:44:42,870] {dag.py:2390} INFO - Creating ORM DAG for tutorial
[2022-12-24 22:44:42,884] {logging_mixin.py:115} INFO - [2022-12-24 22:44:42,884] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:44:42,908] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.231 seconds
[2022-12-24 22:45:13,080] {processor.py:153} INFO - Started process (PID=1702) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:45:13,081] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:45:13,081] {logging_mixin.py:115} INFO - [2022-12-24 22:45:13,081] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:45:13,103] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:45:13,123] {logging_mixin.py:115} INFO - [2022-12-24 22:45:13,123] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:45:13,142] {logging_mixin.py:115} INFO - [2022-12-24 22:45:13,142] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:45:13,166] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.088 seconds
[2022-12-24 22:45:43,329] {processor.py:153} INFO - Started process (PID=1721) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:45:43,330] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:45:43,331] {logging_mixin.py:115} INFO - [2022-12-24 22:45:43,331] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:45:43,356] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:45:43,376] {logging_mixin.py:115} INFO - [2022-12-24 22:45:43,376] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:45:43,393] {logging_mixin.py:115} INFO - [2022-12-24 22:45:43,393] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:45:43,416] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.089 seconds
[2022-12-24 22:46:13,582] {processor.py:153} INFO - Started process (PID=1731) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:46:13,582] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:46:13,583] {logging_mixin.py:115} INFO - [2022-12-24 22:46:13,583] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:46:13,606] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:46:13,626] {logging_mixin.py:115} INFO - [2022-12-24 22:46:13,626] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:46:13,646] {logging_mixin.py:115} INFO - [2022-12-24 22:46:13,646] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:46:13,659] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.079 seconds
[2022-12-24 22:46:43,812] {processor.py:153} INFO - Started process (PID=1741) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:46:43,813] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:46:43,814] {logging_mixin.py:115} INFO - [2022-12-24 22:46:43,814] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:46:43,839] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:46:43,861] {logging_mixin.py:115} INFO - [2022-12-24 22:46:43,860] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:46:43,880] {logging_mixin.py:115} INFO - [2022-12-24 22:46:43,880] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:46:43,905] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.095 seconds
[2022-12-24 22:47:14,070] {processor.py:153} INFO - Started process (PID=1751) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:47:14,071] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:47:14,072] {logging_mixin.py:115} INFO - [2022-12-24 22:47:14,072] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:47:14,095] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:47:14,115] {logging_mixin.py:115} INFO - [2022-12-24 22:47:14,115] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:47:14,134] {logging_mixin.py:115} INFO - [2022-12-24 22:47:14,134] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:47:14,157] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.089 seconds
[2022-12-24 22:47:44,318] {processor.py:153} INFO - Started process (PID=1761) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:47:44,319] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:47:44,320] {logging_mixin.py:115} INFO - [2022-12-24 22:47:44,320] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:47:44,340] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:47:44,359] {logging_mixin.py:115} INFO - [2022-12-24 22:47:44,359] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:47:44,375] {logging_mixin.py:115} INFO - [2022-12-24 22:47:44,375] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:47:44,404] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.087 seconds
[2022-12-24 22:48:14,558] {processor.py:153} INFO - Started process (PID=1771) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:48:14,576] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:48:14,577] {logging_mixin.py:115} INFO - [2022-12-24 22:48:14,577] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:48:14,597] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:48:14,616] {logging_mixin.py:115} INFO - [2022-12-24 22:48:14,616] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:48:14,632] {logging_mixin.py:115} INFO - [2022-12-24 22:48:14,632] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:48:14,649] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.093 seconds
[2022-12-24 22:48:44,795] {processor.py:153} INFO - Started process (PID=1781) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:48:44,796] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:48:44,796] {logging_mixin.py:115} INFO - [2022-12-24 22:48:44,796] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:48:44,816] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:48:44,834] {logging_mixin.py:115} INFO - [2022-12-24 22:48:44,834] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:48:44,850] {logging_mixin.py:115} INFO - [2022-12-24 22:48:44,850] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:48:44,872] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.079 seconds
[2022-12-24 22:49:15,034] {processor.py:153} INFO - Started process (PID=1791) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:49:15,035] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:49:15,035] {logging_mixin.py:115} INFO - [2022-12-24 22:49:15,035] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:49:15,056] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:49:15,074] {logging_mixin.py:115} INFO - [2022-12-24 22:49:15,074] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:49:15,090] {logging_mixin.py:115} INFO - [2022-12-24 22:49:15,090] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:49:15,113] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.081 seconds
[2022-12-24 22:49:45,346] {processor.py:153} INFO - Started process (PID=1801) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:49:45,347] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:49:45,347] {logging_mixin.py:115} INFO - [2022-12-24 22:49:45,347] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:49:45,369] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:49:45,388] {logging_mixin.py:115} INFO - [2022-12-24 22:49:45,388] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:49:45,405] {logging_mixin.py:115} INFO - [2022-12-24 22:49:45,405] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:49:45,430] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.085 seconds
[2022-12-24 22:50:15,586] {processor.py:153} INFO - Started process (PID=1811) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:50:15,587] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:50:15,588] {logging_mixin.py:115} INFO - [2022-12-24 22:50:15,587] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:50:15,610] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:50:15,629] {logging_mixin.py:115} INFO - [2022-12-24 22:50:15,629] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:50:15,646] {logging_mixin.py:115} INFO - [2022-12-24 22:50:15,646] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:50:15,686] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.102 seconds
[2022-12-24 22:50:46,363] {processor.py:153} INFO - Started process (PID=1821) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:50:46,364] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:50:46,365] {logging_mixin.py:115} INFO - [2022-12-24 22:50:46,365] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:50:46,391] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:50:46,410] {logging_mixin.py:115} INFO - [2022-12-24 22:50:46,410] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:50:46,431] {logging_mixin.py:115} INFO - [2022-12-24 22:50:46,431] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:50:46,444] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.083 seconds
[2022-12-24 22:51:16,594] {processor.py:153} INFO - Started process (PID=1831) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:51:16,595] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:51:16,595] {logging_mixin.py:115} INFO - [2022-12-24 22:51:16,595] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:51:16,617] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:51:16,636] {logging_mixin.py:115} INFO - [2022-12-24 22:51:16,636] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:51:16,652] {logging_mixin.py:115} INFO - [2022-12-24 22:51:16,652] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:51:16,663] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.071 seconds
[2022-12-24 22:51:27,666] {processor.py:153} INFO - Started process (PID=1836) to work on /opt/airflow/dags/tutorial.py
[2022-12-24 22:51:27,667] {processor.py:641} INFO - Processing file /opt/airflow/dags/tutorial.py for tasks to queue
[2022-12-24 22:51:27,668] {logging_mixin.py:115} INFO - [2022-12-24 22:51:27,668] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial.py
[2022-12-24 22:51:27,705] {processor.py:651} INFO - DAG(s) dict_keys(['tutorial']) retrieved from /opt/airflow/dags/tutorial.py
[2022-12-24 22:51:27,725] {logging_mixin.py:115} INFO - [2022-12-24 22:51:27,725] {dag.py:2371} INFO - Sync 1 DAGs
[2022-12-24 22:51:27,742] {logging_mixin.py:115} INFO - [2022-12-24 22:51:27,742] {dag.py:2919} INFO - Setting next_dagrun for tutorial to 2022-12-24T00:00:00+00:00, run_after=2022-12-25T00:00:00+00:00
[2022-12-24 22:51:27,758] {processor.py:161} INFO - Processing /opt/airflow/dags/tutorial.py took 0.094 seconds
